operator/cilium_node.go

```diff
@@ -11,7 +11,6 @@ import (
 	v2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2"
```

Comment:
Can you add a `Fixes: 6d44f4cde0f2 ("operator: sync cilium nodes to kvstore instead of k8s nodes")`  in this commit?